src/libs/ajax.js

```diff
@@ -127,6 +127,11 @@ const User = signal => ({
     }
   },
 
+  getPrivacyPolicy: async () => {
+    const response = await fetchSam('privacy/text', _.merge(authOpts(), { signal }))
```

Comment:
Should users be able to view the terms of service before logging in?

Reply:
Good point. They should.

Reply:
tested locally, and can still access the privacy policy even if not authed. Sam doesn't require auth for this new endpoint.